common/c/targets/esp8266/ld/eagle.app.v6.ld

```diff
@@ -160,11 +160,14 @@ SECTIONS
   {
     _irom0_text_start = ABSOLUTE(.);
     *(.irom0.literal .irom.literal .irom.text.literal .irom0.text .irom.text)
-    *libkaac_s.a:(.literal .text .literal.* .text.* .stub .gnu.warning .gnu.linkonce.literal.* .gnu.linkonce.t.*.literal .gnu.linkonce.t.*)
+    *libkaac.a:(.literal .text .literal.* .text.* .stub .gnu.warning .gnu.linkonce.literal.* .gnu.linkonce.t.*.literal .gnu.linkonce.t.*)
```

Comment:
`libkaac` is a static library and therefore it does not include other static libraries.
Regarding `libextension_*` line, the original intention was to put `libkaac` together with part of extensions in `.irom` section while all other extensions will go to `.iram` section. Also, I don't believe it is a good idea to put all the extensions in `.irom0` section.